<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Emp</class>
 <widget class="QDialog" name="Emp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>631</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">GroupBox {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter un employé</string>
    </attribute>
    <widget class="QLineEdit" name="l_CIN">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>112</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_prenom">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_nom">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_salaire">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_tel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fonction:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salaire:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sexe:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Num-tel:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenom:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>112</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QComboBox" name="l_sexe">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
     </property>
     <item>
      <property name="text">
       <string>Homme</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Femme</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="l_Fonction">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
     </property>
     <item>
      <property name="text">
       <string>Administrateur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Présenteur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Montage</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Camera">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>151</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
     </property>
     <property name="text">
      <string>Option cam</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}
</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
     <property name="icon">
      <iconset theme="poubelle">
       <normaloff>../../../../Desktop/Projetcpp</normaloff>../../../../Desktop/Projetcpp</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher un employé</string>
    </attribute>
    <widget class="QTableView" name="tab_etud">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>451</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PDF">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>360</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
     </property>
     <property name="text">
      <string>Generer PDF</string>
     </property>
    </widget>
    <widget class="QLabel" name="background_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>-10</y>
       <width>781</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Desktop/backgroung qt1.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="background_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>781</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Desktop/ground.jpg</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="Imprimer">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>360</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CIN:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_r_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border-radius:9px;
border: 2px solid #000000;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.0, fy: -0.0,
radius: 1.89, stop: 0 #ffffff, stop: 1 #999);
}
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Recherche">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>96</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Tri_cin">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>140</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QRadioButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
     </property>
     <property name="text">
      <string>tri par CIN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Tri_nom">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>180</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QRadioButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
     </property>
     <property name="text">
      <string>tri par nom</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Tri_fonct">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>220</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {
color: #ffffff;
border: 3px solid #010203;
font: 8pt &quot;MS Reference Sans Serif&quot;;
border-radius: 15px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #c4c9c7);
min-width: 80px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 3, stop: 0 #010203, stop: 1 #010203);
}

 QRadioButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 3, stop: 0 #c4c9c7, stop: 1 #c4c9c7);
}</string>
     </property>
     <property name="text">
      <string>tri par fonction</string>
     </property>
    </widget>
    <zorder>background_7</zorder>
    <zorder>background_2</zorder>
    <zorder>tab_etud</zorder>
    <zorder>PDF</zorder>
    <zorder>Imprimer</zorder>
    <zorder>label_10</zorder>
    <zorder>l_r_2</zorder>
    <zorder>Recherche</zorder>
    <zorder>Tri_cin</zorder>
    <zorder>Tri_nom</zorder>
    <zorder>Tri_fonct</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
